docs/workflows/sign-images.rst

```diff
@@ -298,3 +298,27 @@ It is possible to specify a single manifest identified by tag or a list of manif
 by proviging ``tags_list`` option to the call.
 Note that ``manifest lists`` are not signed, instead all the image manifests that manifest lists
 contain, are signed.
+
+Push signed images to the Pulp Registry
+=======================================
+
+Signatures can be pushed to the Pulp Registry via the extensions API. This API exposes an endpoint
+for reading and writing image signatures.
+
+To add a new signature to an image, issue the following PUT request::
+
+    $ http PUT http://localhost:24817/extensions/v2/<namespace>/<name>/signatures/sha256:<manifest-digest> < signature.json
+
+The JSON payload should have the following structure::
```

Comment:
how about referencing  for more info/specs to this source https://github.com/containers/image/blob/main/docs/signature-protocols.md#openshift-dockerdistribution-api-extension